File: backend/src/main/java/com/bookbook/domain/user/dto/UserCreateRequestDto.java
package com.bookbook.domain.user.dto;

import jakarta.validation.constraints.NotBlank;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
public class UserCreateRequestDto {
    @NotBlank(message="아이디는 필수 입력값입니다.")
    private String nickname;
    @NotBlank(message="주소는 필수 입력값입니다.")
    private String address;
}

File: backend/src/main/java/com/bookbook/domain/user/dto/request/UserCreateRequestDto.kt
package com.bookbook.domain.user.dto.request

import jakarta.validation.constraints.NotBlank

data class UserCreateRequestDto(
    @field:NotBlank(message = "아이디는 필수 입력값입니다.")
    var nickname: String,
    @field:NotBlank(message = "주소는 필수 입력값입니다.")
    var address: String
)